### ISSUE TITLE
Multiple Coda connection support

### ISSUE THREAD

**Issue by svennu** [parker-codes/coda-js#7]
Hi,

I love the library, helps nicely to connect to Coda DB. Thank you!

I've noticed that I could not use multiple Coda API tokens at the same time. The issue seems to be due to the usage of `axios.defaults.*` in the `API` constructor. And if my tests are correct, these even override the `axios` if used outside of the lib :(

Anyways, proposing to implement [instance-based axios initialisation](https://github.com/axios/axios#creating-an-instance). For example, class `API`'s constructor would be something like:
```
  [...]
  private _axiosInstance: any;

  constructor(token: string) {
    this._axiosInstance = axios.create({
      baseURL: 'https://coda.io/apis/v1beta1',
      headers: {'Authorization': `Bearer ${token}`}
    });
  }
 [...]
```
Seems to be working fine. What do you think, can you move it into the official library? Cheers!

**Comment by parker-codes** [comment]
I think that is a great idea! Especially if it solves the override problem.

Would you like to make this update via a PR? I'd be happy to walk you through it if you'd like.

**Comment by svennu** [comment]
I've tested my code on my services, seems to be all good. Happy to provide PR.

Any random naming for the PR would work? Or you have some more concrete guidelines?

Also shall I include `microbundle` output too, or you could do it as part of npm publish?

++ a bump in the axios library to [0.19.2](https://github.com/axios/axios/blob/master/CHANGELOG.md) could also be in order, while we're at it 😄 ?

**Comment by parker-codes** [comment]
No concrete guidelines for naming the PR, but probably not random. 😅You could name it the same as this issue.

I'll take care of the microbundle part, but you could bump the Axios version. They've been pretty stable.

**Comment by svennu** [comment]
Allright,
Ready for review: https://github.com/parker-codes/coda-js/pull/8

**Comment by parker-codes** [comment]
@svennu Thanks again for contributing! This was released in **v2.3.0**!

**Comment by svennu** [comment]
Excellent. Can take your release into use. Cheers!
